operator/pkg/controllers/master/master.go

```diff
@@ -60,6 +60,8 @@ func (c *Controller) Reconcile(ctx context.Context, controlPlane *v1alpha1.Contr
 		c.reconcileScheduler,
 		c.reconcileAuthenticator,
 		c.iamController.Reconcile,
+		c.reconcileEncryptionProviderConfig,
```

Comment:
I'd like to (eventually) migrate to making these objects instead of methods. Let's punt for now.
```suggestion
		c.encryptionProvider,
```

Reply:
We have an `reconcileEncryptionProviderConfig` and `reconcileEncryptionProvider`, won't just having `encryptionProvider` be confusing which objects/resource is reconciler